File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoFormula.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Symbol;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdFormula;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoFormula extends StudioBase
{
  public MetaIdField assignToFieldId;

  @Nullable
  public StudioValueVarIdCondition conditionVarId;

  @Nullable
  public StudioValueCodeJavascript formula;

  public MetaIdFormula metaId;

  @Nullable
  public Symbol name;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoFormula.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdFormula
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioValueCodeJavascript
import com.neome.api.meta.base.dto.StudioValueVarIdCondition
import com.neome.api.meta.base.Symbol

interface StudioDtoFormula : StudioBase
{
  val assignToFieldId: MetaIdField
  val conditionVarId: StudioValueVarIdCondition?
  val formula: StudioValueCodeJavascript?
  val metaId: MetaIdFormula
  val name: Symbol?
}